scripts/r.reclass.area/r.reclass.area.py

```diff
@@ -163,18 +168,29 @@ def reclass(inf, outf, lim, clump, diag, les):
     p1.wait()
     p2.stdin.close()
     p2.wait()
+    expression = "$outfile = $recfile"
+    mapcalc_kwargs = {"outfile": outfile, "recfile": recfile}
     if p2.returncode != 0:
         if lesser:
-            grass.fatal(
-                _("No areas of size less than or equal to %f " "hectares found.")
-                % limit
+            op_str = "less"
+        else:
+            op_str = "greater"
```

Comment:
The original code is right. Sentences cannot be build from parts because that makes translations impossible. If really needed, you could create two sentences and then put them together using another, possibly translatable, string.